src/proxy-ui-api/frontend/src/locales/en.json

```diff
@@ -172,6 +174,8 @@
     "removeAll": "Remove All",
     "save": "Save",
     "continue": "Continue",
+    "activate": "Activate",
+    "deactivate": "Disable",
```

Comment:
Why is this key named `deactivate` (not `disable`), when the related success message is `disableSuccess`? 

Are localization keys named after UI texts or API terms? Picking one consistent naming convention would improve readability.

Reply:
The term used in openapi-definition is deactivate while disable is used in the task description. This is the only place 'disable' is used. Simultaneously another task uses the term 'activate', not 'enable'. 

Not sure whether all deactive -words shoud be changed to disable or the other way around

Reply:
Also if we use 'disable' then in addition to refactoring deactivate -> disable in api, should we refactor activate -> enable?

Reply:
Using activate/deactive is one word change and using disable/enable needs change in multiple files (not a complicated change though)

Reply:
I would not change the API easily as it can be quite a lot of work if you need to make the change consistently across controller + services + helpers etc. The terms in API usually reflect the core classes and e.g. SignerProxy. If some term is found to be unquestionably bad, ok then, but not just because of this localization key issue. 

I don't think the terms in UI and API need to be necessarily the same. It would be obviously nicer if they were, but sometimes the terms in core classes are a bit too technical and UI prefers different terms. And having the whole backend (api + proxy-ui-api + core) speak same language is better than having (ui + api) speak one language and (core) speak a different one, IMO.

If we look at this single `en.json` file then in any case I guess it should be internally consistent? So either

- `deactivate` and `deactivateSuccess` OR
- `disable` and `disableSuccess`

?

Reply:
we have agreed to use API terms in the keys